doc/release/README.chplenv

```diff
@@ -341,17 +341,20 @@ Optional Settings
 
 *  Optionally, the CHPL_HWLOC environment variable can select between
    no hwloc support or using the hwloc package distributed with Chapel
-   in third-party.  Current options are:
+   in third-party.  Note that hwloc is only used by the qthreads
+   tasking layer, and does not need to be built for other tasking
+   layers.  Current options are:
 
        none   : do not build hwloc support into the Chapel runtime
        hwloc  : use the hwloc distribution bundled with Chapel in
                 third-party
 
    If unset, CHPL_HWLOC defaults to "hwloc" if CHPL_TASKS is qthreads,
-   unless the target platform is knc.  In all other cases it defaults to
-   "none".  Note that the bundled hwloc distribution will not build on
-   some targets (Mac, for example), but the CHPL_HWLOC default does not
-   reflect that.
+   unless the target platform is knc.  In all other cases it defaults
+   to "none".  If the bundled hwloc distribution does not build
+   successfully, it should still be possible to use qthreads so long
+   as CHPL_LOCALE_MODEL is flat.  Manually set CHPL_HWLOC to "none" and
```

Comment:
I think things will work functionally w/o hwloc for numa (so long as qthreads finds libnuma.) We end up with 1 sub-locale per core instead of per numa domain. I'd guess that this will really kill performance since the work stealing issue is going to be heavily exaggerated. 

I know we get almost identical performance in our nightly testing with qthreads+flat w/o hwloc on chap04 so I wasn't bothered recommending this if hwloc didn't built. But it seems hard to recommend this for numa. What do you think?